pkg/driver/node_linux.go

```diff
@@ -116,6 +125,31 @@ func (d *nodeService) findDevicePath(devicePath, volumeID, partition string) (st
 	return canonicalDevicePath, nil
 }
 
+func checkVolume(canonicalDevicePath, strippedVolumeName string) error {
```

Comment:
np: Readability can be improved here by using a more descriptive func name, perhaps`verifyVolumeSerialMatch`?